tests/python-pytest/onnx/import/test_cases.py

```diff
@@ -19,7 +19,6 @@
 
 IMPLEMENTED_OPERATORS_TEST = [
     'test_split_equal'
-    'test_random_',
```

Comment:
why remove this?

Reply:
there seems to be no test for test_random_ in the onnx repo